### PR TITLE
Feature/portal optimize

### PR DESCRIPTION
**Pull request by ihannache**
# FIX | FEAT | REFACTOR | PERF | DOCS : TITLE

- closes #TICKET_NUMBER

## Description

[JIRA LINK]

Acceptance Criterias

## Validation

- [ ] Code Approved
- [ ] QA Done
- [ ] Design/UI Approved from design

## QA

Steps to validate
Url (storybook, ...)
...

## Mention

@kstonge @luclemo

### CODE REVIEW

File: src/global.d.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+declare global {
```
Comment (atoulous) [inline src/global.d.ts:1]: ce fichier peut etre mis dans `src/helpers` ?

File: package.json [hunk 2]
```diff
@@ -138,6 +138,7 @@
     "prettier": "^2.0.5",
     "prettier-eslint": "^10.1.1",
     "typescript": "^4.4.4",
-    "web-vitals": "^2.1.2"
+    "web-vitals": "^2.1.2",
+    "@babel/plugin-proposal-private-property-in-object": "^7.14.5"
```
Comment (sebastienbonami) [inline package.json:142]: This package has been deprecated, you should replace it by `@babel/plugin-transform-private-property-in-object`.

**Review by sebastienbonami — CHANGES_REQUESTED** [review]

**Comment by sebastienbonami** [comment]
It's not that important, but the main comment of this PR should be edited has it's still a template.

**Comment by sebastienbonami** [comment]
The check "Quality / Install Dependencies" fails because `package.json` and `package-lock.json` are not synced anymore. To fix.

File: entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```
Comment (atoulous) [inline entrypoint.sh:1]: Pour éviter de re-lister les variables ici j'ai réfléchi à importer les variables du file EnvVariables.ts, mais sans succes car c'est un shell file. Faudrait Si tu as une idée..

**Pull request closed without merging**